plugins/experimental/webp_transform/Makefile.inc

```diff
@@ -25,8 +25,8 @@ experimental_webp_transform_WebpTransform_la_SOURCES = \
 
 experimental_webp_transform_WebpTransform_la_LDFLAGS = \
   $(AM_LDFLAGS) \
-  -L$(top_builddir)/lib/cppapi
+  -L$(top_builddir)/tscpp/cpp
```

Comment:
This should be:

` -L$(top_builddir)/include/tscpp/api`

Reply:
No, it should be `-L$(top_builddir)/src/tscpp/api/`. It's a library linking path, not an include path. It should be the same as the `libtscppapi` value in `plugins/Makefile.am`.

Linux must have very relaxed linking requirements - AFAICT this was broken for the example plugins yet checking my local standard and out of tree build runs, the plugin `.so` files are there for plugins, experimental plugins, and example plugins. Looking at the CI, I'm not sure any of them actually build those plugins. We should have at least one CI build that builds all of the plugins.